lib/protocol/networks.ts

```diff
@@ -80,8 +80,7 @@ async function getLocalNetworkConfig(network: string, source: string): Promise<P
     locator: config["lidoLocator"].proxy.address,
     agentAddress: config["app:aragon-agent"].proxy.address,
     votingAddress: config["app:aragon-voting"].proxy.address,
-    easyTrackAddress: config["app:aragon-agent"].proxy.address,
-    sdvt: config["app:node-operators-registry"].proxy.address,
```

Comment:
why remove it here?

Reply:
`easyTrackAddress` goes to `app: Aragon-voting,` and sdvt now is taken directly from deployment artifacts. No need to link NOR manually anymore.